api/unstable/renegotiate.h

```diff
@@ -110,6 +110,7 @@ S2N_API int s2n_config_set_renegotiate_request_cb(struct s2n_config *config, s2n
  *
  * @note This method MUST be called before s2n_renegotiate.
  * @note Calling this method on a server connection will fail. s2n-tls servers do not support renegotiation.
+ * @note This method will fail if the connection has indicated that it will be serialized.
  *
```

Comment:
Reference the config method that'll cause it to fail. Since the method isn't like "s2n_config_indicate_will_serialize" it's not really obvious.